src/IntegrationUtils.ts

```diff
@@ -464,12 +457,7 @@ export function computeKamelArgs(absoluteRoot: string,
 		kamelArgs.push(`--profile=${profile}`);
 	}
 	if (resource && resource.trim().length > 0) {
-		let resourceArray = resource.split(' ');
-		if (resourceArray && resourceArray.length > 0) {
-			resourceArray.forEach(res => {
-				kamelArgs.push(`--resource="${res}"`);
-			});
-		}
+		kamelArgs.push(`--resource=${resource}`);
```

Comment:
2 things here:
* remove double-quotes which are nto working. The spaces are handled correctly by using different parameters passed to the child_process
* remove the support of the multiple resources. Space character was used a seprator inside a string. it doesn't work when there is a space in the path of one of the resource. Given the previous problem of double-quotes, it has never worked anyway, so better removing for now to simplify code and introduce in another PR the support of multiple resources using a data structure instead of merging/splitting strings.

Reply:
Don't we need the double quotes for any resources with spaces in the path?

Reply:
the test https://github.com/camel-tooling/vscode-camelk/pull/823/files#diff-b7ed3e6f1115050b03813503cf26ba0ef985fc1ca54ef08e59f581d50c55f9e6R78 is proving that we do not need them (at least on Linux as we do not have Windows CI with kamel instance). In fact, it was the root cause of non-working `Start integration with resource` command.

Based on https://nodejs.org/api/child_process.html#child_process_child_process_exec_command_options_callback, the double-quotes would be needed only for program path.

```
exec('"/path/to/test file/test.sh" arg1 arg2');
// Double quotes are used so that the space in the path is not interpreted as
// a delimiter of multiple arguments.
```

For the arguments, it is not mentioned that we need to escape any characters.